applications/GeoMechanicsApplication/custom_elements/transient_Pw_element.hpp

```diff
@@ -119,17 +119,13 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) TransientPwElement : public UPwSmall
     std::string Info() const override
     {
         std::stringstream buffer;
```

Comment:
@rfaasse: This one has been missed, since it has been done properly elsewhere. Local variable `buffer` is no longer used, so it can be removed.

Reply:
Good catch, removed